src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -717,8 +717,12 @@ def setUp(self):
                 self.hostfile_clients_slots)
 
         # Access points to use by default when starting servers and agents
-        self.access_points = self.params.get(
-            "access_points", "/run/setup/*", list(self.hostlist_servers)[:1])
+        #  - for 1 or 2 servers use 1 access point
+        #  - for 3 or more servers use 3 access points
+        access_points_qty = 1 if len(self.hostlist_servers) < 3 else 3
+        default_access_points = self.hostlist_servers[:access_points_qty]
+        self.access_points = NodeSet(
+            self.params.get("access_points", "/run/setup/*", default_access_points))
```

Comment:
Why not use 2 access points with 2 servers?

Reply:
This was Johann's recommendation